cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1421,6 +1439,69 @@ def test_create_non_deprecated_problems(self):
         template_display_names = [template['display_name'] for template in templates[0]['templates']]
         self.assertEqual(template_display_names, ['Annotation', 'Poll', 'Survey'])
 
+    def test_deprecated_no_advance_component_button_with_admin_settings(self):
+        """
+        Test that there will be no `Advanced` button on unit page if units are
+        deprecated from the admin settings `XBlockDeprecatedAdvancedComponentConfig`
+        provided that they are the only modules in `Advanced Module List`
+        """
+        model = XBlockDeprecatedAdvancedComponentConfig.objects.create(disabled_blocks='poll survey', enabled=True)
+        model.save()
```

Comment:
`create` does save the object so no need for this.

https://docs.djangoproject.com/en/1.8/ref/models/querysets/#create